Answers.md

```diff
@@ -0,0 +1,9 @@
+1. Adaptive websites have different plans for different sizes, i.e mobile, tablet. Fully responsive websites adjust to any size without different plans. 
+
+2. Mobile first may take longer due to adjusting the sizes and outline for the program. Desktop first is more detailed due to being designed for a large screen. 
+
+3. In rem units, it would cause the font-size to be written lower than what is provided. For example, in rem units, it would be written as font-size: 3.5rem instead of 62.5.
```

Comment:
It's mostly to make conversion between rem and px easier to decipher. With 62.5% set in the font size, 1.6 rem would be equivalent to 16px.